src/main/java/com/google/summit/ast/statement/LoopStatement.kt

```diff
@@ -32,40 +32,40 @@ sealed class LoopStatement(val body: Statement, loc: SourceLocation) : Statement
 /**
  * A traditional three-part `for` loop statement.
  *
- * @property declarations the variables declared at the start
+ * @property declarationGroup the variables declared at the start
  * @property initializations the expressions initialized at the start
  * @property updates the expressions evaluated on each iteration
  * @property condition the optional loop condition
  * @param body the statement being looped
  * @param loc the location in the source file
  */
 class ForLoopStatement(
-  val declarations: List<VariableDeclaration>,
+  val declarationGroup: VariableDeclarationGroup?,
   val initializations: List<Expression>,
   val updates: List<Expression>,
   val condition: Expression?,
   body: Statement,
   loc: SourceLocation
 ) : LoopStatement(body, loc) {
   override fun getChildren(): List<Node> =
-    declarations + initializations + listOfNotNull(condition, body) + updates
+    initializations + listOfNotNull(declarationGroup, condition, body) + updates
 }
 
 /**
  * A enhanced `for` loop statement.
  *
- * @property elementDeclaration the element iterator
+ * @property elementDeclarations the element iterator
  * @property collection the collection to iterate on
  * @param body the statement being looped
  * @param loc the location in the source file
  */
 class EnhancedForLoopStatement(
-  val elementDeclaration: VariableDeclaration,
+  val elementDeclarations: VariableDeclarationGroup,
```

Comment:
Here and elsewhere: I think it would be better to name this consistently with the class name, i.e. `elementDeclarationGroup`

Reply:
Renamed properties to avoid partial or plural variants of `VariableDeclarationGroup`.